benchopt/base.py

```diff
@@ -420,7 +420,7 @@ def evaluate_result(self, **solver_result):
 
         Returns
         -------
-        objective_value : float or dict {'name': float}
+        objective_value : float or dict {'name': float} or list of dict
```

Comment:
btw this may be interpreted as: the dict must have a single key whose name is `'name'`

Reply:
```suggestion
        objective_value : float or dict {str: float} or list of dict
```